.semversioner/next-release/patch-20241008011651057484.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "patch",
```

Comment:
Should we bump this to minor so the next release containing incremental become 0.3.7?

Reply:
Will be bumped to minor on the big merge :)